src/win/sugar.rs

```diff
@@ -1,16 +1,16 @@
 use std::{fmt, mem::MaybeUninit};
 
-use ntapi::ntrtl::{RtlInitUnicodeStringEx, RtlNtStatusToDosError};
-use winapi::shared::ntdef::NT_SUCCESS;
-use windows_sys::Win32::Foundation::{NTSTATUS, UNICODE_STRING};
+use windows_sys::Win32::Foundation::{RtlNtStatusToDosError, NTSTATUS, UNICODE_STRING};
+
+use super::windows_sys_gap_defs::init_unicode_string;
 
 pub struct NTStatusError {
     pub status: NTSTATUS,
 }
 
 impl NTStatusError {
     pub fn from(status: NTSTATUS) -> std::result::Result<(), NTStatusError> {
-        if NT_SUCCESS(status) {
+        if status >= 0 {
```

Comment:
Just a thought, perhaps we should define a fn NT_STATUS; to help make the translation of this to the equivalent C code more obvious?

Reply:
Done